srk-nixpkgs.nix

```diff
@@ -1,189 +1,54 @@
 { pkgs ? (import <nixpkgs> {})
 , compiler ? pkgs.haskell.packages.ghc802
-, genesisN ? 3 
-, slotDuration ? 20 
-, networkDiameter ? 6
-, mpcRelayInterval ? 16 } :
```

Comment:
This is long time not used anymore. One can now just fork cardano-sl and change the yaml file. It's not much different than changing it from Nix.

I don't know how Phillip is doing it now, but this doesn't work.